great_expectations/expectations/expectation.py

```diff
@@ -338,7 +344,7 @@ class Expectation(metaclass=MetaExpectation):
         "result_format",
     )
     default_kwarg_values: ClassVar[
-        dict[str, bool | str | float | RuleBasedProfilerConfig | None]
+        dict[str, Union[bool, str, float, RuleBasedProfilerConfig, None]]
```

Comment:
All these typing changes are to appease Pydantic's internal logic around parsing types